docs/root/intro/version_history.rst

```diff
@@ -42,6 +42,7 @@ Version history
 * http: remove h2c upgrade headers for HTTP/1 as h2c upgrades are currently not supported.
 * http: absolute URL support is now on by default. The prior behavior can be reinstated by setting :ref:`allow_absolute_url <envoy_api_field_core.Http1ProtocolOptions.allow_absolute_url>` to false.
 * http: support :ref:`host rewrite <envoy_api_msg_config.filter.http.dynamic_forward_proxy.v2alpha.PerRouteConfig>` in the dynamic forward proxy.
+* http: support :ref:`disable filter per route <envoy_api_msg_config.filter.http.grpc_http1_reverse_bridge.v2alpha1.FilterConfigPerRoute>` in the grpc http1 reverse bridge filter.
```

Comment:
grammar nit: support disabling the filter per route

Reply:
Fixed in https://github.com/envoyproxy/envoy/pull/8645/commits/0edac0420944a19891d43804acbe6fd3f0edf654